docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "2.1"
+
+services:
+
+  osqa:
+    build:
+      context: .
+    environment:
+      OSQA_DB_USER: osqa
+      OSQA_DB_PASSWORD: osqa
+      OSQA_DB_NAME: osqa
+      OSQA_DB_HOST: postgres
+
+    ports:
+      - 8080:8080
+    depends_on:
+      postgres:
+        condition: service_healthy
+      memcached:
+        condition: service_started
```

Comment:
Do we want to run these in separate containers? I think for discourse we put the database on the host didn't we?

Reply:
Docker the best practise is one task one container. The `docker-compose.yml` is only an example used locally for testing.